mdsuite/visualizer/znvis_visualizer.py

```diff
@@ -116,12 +116,10 @@ def _prepare_species(self):
         particle_list = []
         for item in self.species:
             colour, radius = self._get_species_properties(item)
-            trajectory = np.transpose(
-                self.database.load_data(
-                    [join_path(item, self.identifier)], select_slice=np.s_[:]
-                )[join_path(item, self.identifier)],
-                axes=[1, 0, 2],
-            )
+            trajectory = self.database.load_data(
+                [join_path(item, self.identifier)], select_slice=np.s_[:]
+            )[join_path(item, self.identifier)]
+            trajectory = np.transpose(trajectory, axes=[1, 0, 2])
```

Comment:
I find this hard to read, maybe split it up and remove the selected_slice because it is already the default:

```py
trajectory = self.database.load_data([join_path(item, self.identifier)])
trajectory = trajectory[join_path(item, self.identifier)]
trajectory = np.transpose(trajectory, axes=[1, 0, 2])
```

Reply:
Yep will do

Reply:
Resolved in 413096d